static/backbone/barchart.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Rosa Parks Bar Chart</title>
+		<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet">
+        <link href="http://netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.css" rel="stylesheet">
+		<link rel="stylesheet" type="text/css" href="/static/backbone/css/lg.css">
+		<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/backbone-forms/0.14.0/templates/bootstrap3.css">
+		<style>
+			body {
+				overflow: hidden;
+			}
+            .chart {
+                width: 58%;
+                display: inline-block;
+                height: 400px;
+                margin-left: 5px;
+                border: solid 1px #CCC;
+                margin-top: 20px;
+            }
+            #chart1 {
+                width: 30%;
+            }
+            .container-fluid {
+                width: 92%;
+                margin-left: auto;
+                margin-right: auto;
+            }
+		</style>
+	</head>
+	<body>
+		<!-- Header -->
+		<nav id="navbar" class="navbar navbar-default" role="navigation">
+            <ul class="nav navbar-nav">
+                <li><a href="#">Local Ground</a></li>
+                <li id="variable_container">
+                    <select id="variable_select" class="form-control" style="margin-top:7px;">
+                        <option value="-1">-- Select an Indicator --</option>
+                    </select>
+                </li>
+			</ul>
+        </nav>
+        <div class="container-fluid">
+            <h1>Why Are There Worms in Some Places and Not in Others?</h1>
+            <div id="charts">
+                <div id="chart1" class="chart"></div>
+                <div id="chart2" class="chart"></div>
+            </div>
+        </div>
+	</body>
+    
+    <script src="/static/backbone/js/external/require.js" type="text/javascript"></script>
+    <script src="/static/backbone/js/api/base.js" type="text/javascript"></script>
+    <script>
+        require(
+            ["jquery", "views/charts/barchart"],
+            function($, BarChart) {
+                $(function() {
+                    var username ="vanwars",
+                        password = "my_password";
+                    $.ajaxSetup({
+                        beforeSend: function(xhr){
+                            xhr.setRequestHeader("Authorization",
+                                "Basic " + btoa(username + ":" + password));
+                        }
+                    });
+                    new BarChart();
+                });
+            }
+        );
+    </script>
+</html>
```

Comment:
I'm assuming at some point this is gonna get separated out into separate js/html/css if we are keeping it around?